snapshot/policy/scheduling_policy.go

```diff
@@ -162,7 +165,11 @@ func (p *SchedulingPolicy) checkMissedSnapshot(now, previousSnapshotTime, nextSn
 
 	const halfhour = 30 * time.Minute
 
-	return (len(p.TimesOfDay) > 0 || len(p.Cron) > 0) && p.RunMissed && previousSnapshotTime.Add(oneDay-halfhour).Before(now) && nextSnapshotTime.After(now.Add(halfhour))
+	if !p.RunMissed.OrDefault(false) {
```

Comment:
nit: should this be the following for consistency?
That is, `p.RunMissed` will get the _default_ value when not set?

```suggestion
	if !p.RunMissed.OrDefault(defaultRunMissed) {
```

Reply:
This was done on purpose for the #3330 PR feedback.  Basically by using `OrDefault(false)` we ensure that if a user is upgrading a repo that doesn't have --run-missed that it will never be enabled, regardless of whether the default is true or false.

If you want me to make the change, please confirm and I will, but I think there is a difference between the default for a newly created repo, and the default for an existing repo.

Reply:
I guess it depends on what we decide based on the discussion on #3330.